deployment/centos-package-x64/docker-build.sh

```diff
@@ -8,17 +8,9 @@ set -o xtrace
 # Move to source directory
 pushd ${SOURCE_DIR}
 
-# Prepare the source
-source "$HOME/.nvm/nvm.sh"
-nvm use v8
+# Build RPM
 make -f .copr/Makefile srpm outdir=/root/rpmbuild/SRPMS
-
-# Remove dep for nodejs/yarn since our build env won't have these (NVM instead)
-sed -i '/BuildRequires:  nodejs >= 8 yarn/d' SPECS/jellyfin.spec
-
-# Build the RPMs
-rpmbuild -bs SPECS/jellyfin.spec --define "_sourcedir ${SOURCE_DIR}/SOURCES/pkg-src/"
-rpmbuild -bb SPECS/jellyfin.spec --define "_sourcedir ${SOURCE_DIR}/SOURCES/pkg-src/"
+rpmbuild --recompile /root/rpmbuild/SRPMS/jellyfin-*.src.rpm
```

Comment:
I generally use `rpmbuild --rebuild -bb <src_rpm>` here.  I've never used `--recompile`.  Is it producing an actual binary RPM?  The [docs](http://ftp.rpm.org/max-rpm/s1-rpm-b-command-other-build-commands.html) suggest to me that it doesn't.

Reply:
Yes it seems to build a actual binary RPM, i intended to use the same `rpmbuild -rb` as with Fedora but the CentOS version didn't have it so used the first best from the `--help` text.

Reply:
I replaced the flag with your's since it actually builds a binary RPM.